zaczytanie_nowych_danych

```diff
@@ -0,0 +1,1675 @@
+{
```

Comment:
Ten plik nie jest w poprawnej ścieżce

Reply:
to plik Emilki, ale nie wiem czy on jest w ogóle potrzebny, podpytam Emilki.